test/applications/test_bin_packing.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test bin packing class"""
+from test.optimization_test_case import QiskitOptimizationTestCase
+
+from qiskit_optimization import QuadraticProgram
+from qiskit_optimization.algorithms import OptimizationResult, OptimizationResultStatus
+from qiskit_optimization.applications.bin_packing import BinPacking
+from qiskit_optimization.problems import Constraint, QuadraticObjective, VarType
+
+
+class TestBinPacking(QiskitOptimizationTestCase):
```

Comment:
Can a test be added that covers max_number_of_bins i.e a non-default value for this.